.github/workflows/release-charts.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: |
           set -euo pipefail
           export TAG=$(make version)
-          helm package ./deploy/manifests/controller/helm/retina --version $TAG
+          helm package ./deploy/legacy/manifests/controller/helm/retina --version $TAG
```

Comment:
So we are still using legacy charts? Will we eventually merge the hubble charts with the legacy charts? What is the strategy for this